backend/src/main/java/org/sefglobal/core/controller/MultiverseController.java

```diff
@@ -62,8 +66,11 @@ public University updateUniversity(@PathVariable long id,
     //todo: Implement an endpoint to delete university (change active status)
 
     @GetMapping("/ambassadors")
-    public Iterable<Ambassador> listAmbassador() {
-        return ambassadorRepository.findAll();
+    public Iterable<Ambassador> listAmbassador(@RequestParam(defaultValue = "0",
```

Comment:
The return type should be changed.

Reply:
Can you explain?

Reply:
`ambassadorRepository.findAll` 's return type is `Page<Ambassador>`

Reply:
You have to return a Page